META.json

```diff
@@ -2,15 +2,15 @@
    "name": "citus",
    "abstract": "Scalable PostgreSQL for real-time workloads",
    "description": "Citus horizontally scales PostgreSQL across commodity servers using sharding and replication. Its query engine parallelizes incoming SQL queries across these servers to enable real-time responses on large datasets.",
-   "version": "9.0.1",
+   "version": "9.1.0",
    "maintainer": "\"Citus Data\" <packaging@citusdata.com>",
    "license": "agpl_3",
    "provides": {
       "citus": {
          "abstract": "Citus Distributed Database",
-         "file": "src/backend/distributed/citus--9.0-1.sql",
+         "file": "src/backend/distributed/citus--9.1-0.sql",
```

Comment:
It can be worth checking PGXN specs at https://pgxn.org/spec/#provides :

> file:
The value must contain a relative file path from the root of the distribution to the file containing the extension. The path must be specified with unix conventions. Required.

Reply:
done in 505e630cf071e3c43db6366fd4655431aa16dcbf